px-data-grid-string-renderer.html

```diff
@@ -4,6 +4,31 @@
 <link rel="import" href="px-data-grid-theme.html">
 <link rel="import" href="css/px-data-grid-string-renderer-styles.html">
 
+<!--
+
+### Styling
+The following custom properties are available for styling of the string-renderer content.
+
+Custom property | Description
+'--px-data-grid-cell-string-column-overflow' | controls the overflow behavior, visible, hidden
+'--px-data-grid-cell-string-column-white-space' | controls how white-spaces are handled, normal, nowrap
+'--px-data-grid-cell-string-column-word-wrap' | controls how word-wrapping is handled normal, break-word
+'--px-data-grid-cell-string-column-text-overflow' | controls how text-overflow is handled, clip, ellipsis
+'--px-data-grid-cell-string-column-editor-align-items | control editor alignment when in edit mode, center, flex-end, flex-start'
+
+Examples:
+Make string-renderer columns wrap the content:
+--px-data-grid-cell-string-column-overflow: hidden;
```

Comment:
IMO, <style> wrapper is needed here, i.e:

Make string-renderer columns wrap the content:
```
<style>
 px-data-grid {
  --px-data-grid-cell-string-column-overflow: hidden;
  --px-data-grid-cell-string-column-word-wrap: break-word;
 }
</style>
```

Reply:
This is just the documentation of the css-variables, the syntax is copied from here https://github.com/PredixDev/px-dropdown/blob/master/px-dropdown.html under the style-section.